index.bs

```diff
@@ -933,6 +935,35 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Platform Authenticator Availability - PublicKeyCredential's `isPlatformAuthenticatorAvailable()` method ### {#isPlatformAuthenticatorAvailable}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorAvailable">
+
+[=[RPS]=] use this method to determine whether they can create a new credential using a [=platform authenticator=].
+Upon invocation, the [=client=] employs a platform-specific procedure to discover available [=platform authenticators=].
+If successful, the [=client=] then assesses whether the user is willing to create a credential using one of the available [=platform authenticators=].
+This assessment may include various factors, such as:
+    - Whether the user is running in private or incognito mode.
+    - Whether the user has configured the [=client=] to not create such credentials.
+    - Whether the user has previously expressed an unwillingness to create a new credential for this [=[RP]=],
+        either through configuration or by declining a user interface prompt.
+    - The user's explicitly stated intentions, determined through user interaction.
+If this assessment is affirmative, the promise is resolved with the value of <i>True</i>.
+Otherwise, the promise is resolved with the value of <i>False</i>.
```

Comment:
A few small questions:

1.  If the RP is expected to act only in the case of a positive response, then it's not clear that providing a negative response has any impact over and above the fingerprinting risk. I'd suggest that we could simply not send a negative response (e.g. not resolve the `Promise`).

2.  Is the RP's response to a positive response defined? Talking to @kpaulh, it's not clear that there's agreement between vendors about what question we'd be asking users, and what the response to that question would actually mean. I worry a bit about the UI we'd present in that case, and what we'd actually ask users to approve.

3.  Given the amount of time folks generally spend on login pages, it seems like a delay long enough to be meaningful would be functionally indistinguishable from just not resolving the promise. What kind of timeout were you considering, @kpaulh? (On this point, it's not really clear to me why it would be bad for the promise not to resolve... That's totally something that can happen, and that callers need to deal with (e.g. `return new Promise(_ => {});`).